### PR TITLE
Adds meta info to package.json

### PR DESCRIPTION
**Pull request by joeyklee**
Includes version, name, description. We can better track releases if we end up doing updates, etc.

### CODE REVIEW

**Comment by joeyklee** [comment]
Note to check the license - I put the `cc-by-3.0` matching what the current license is.

File: package.json [hunk 1]
```diff
@@ -1,4 +1,18 @@
 {
+  "version":"2.0",
+  "name": "noc-book-2",
+  "description": "Repository for The Nature of Code book with p5.js",
+  "author": "shiffman",
```
Comment (shiffman) [inline package.json]: full name?

```suggestion
  "author": "Daniel Shiffman",
```

File: package.json [hunk 2]
```diff
@@ -1,4 +1,18 @@
 {
+  "version":"2.0",
+  "name": "noc-book-2",
+  "description": "Repository for The Nature of Code book with p5.js",
```
Comment (shiffman) [inline package.json]:
```suggestion
  "description": "Repository for the book The Nature of Code: 2nd edition",
```

File: package.json [hunk 3]
```diff
@@ -1,4 +1,18 @@
...
+  "author": "shiffman",
+  "license": "cc-by-3.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nature-of-code/noc-book-2.git"
+  },
+  "keywords": ["nature of code", "creative coding", "magicbook", "p5"],
```
Comment (shiffman) [inline package.json]:
```suggestion
  "keywords": ["nature of code", "creative coding", "magicbook", "p5.js"],
```

**Review by shiffman — APPROVED** [review]
Thank you! (some small, trivial comments). A reminder that I might do a license FAQ since the book text and illustrations are licensed differently than the code examples.

**Pull request merged**